Project16-Maps/ViewController.swift

```diff
@@ -6,14 +6,82 @@
 //
 
 import UIKit
+import MapKit
 
-class ViewController: UIViewController {
-
+class ViewController: UIViewController, MKMapViewDelegate {
```

Comment:
To organize better your code you could create an extension of `ViewController` to conform with `MKMapViewDelegate` and to contain all related methods.